src/imports.rs

```diff
@@ -259,7 +259,23 @@ impl UseTree {
             let lo = attrs.last().as_ref()?.span().hi();
             let hi = self.span.lo();
             let span = mk_sp(lo, hi);
-            combine_strs_with_missing_comments(context, &attr_str, &use_str, span, shape, false)
+
+            let allow_extend = if context.config.version() == Version::Two {
+                let line_len = attr_str.len() + 1 + use_str.len();
```

Comment:
What happens if you have several attributes within the threshold ?

For example:

```rust
// rustfmt-inline_attribute_width: 80

#[cfg(feature = "alloc1")]
#[cfg(feature = "alloc2")]
use total_len_is::_50__;
```

Reply:
https://github.com/rust-lang/rustfmt/pull/3409#discussion_r258552901